OpenRA.Mods.Common/Projectiles/NukeLaunch.cs

```diff
@@ -104,6 +106,12 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 			return anim.Render(pos, wr.Palette(weaponPalette));
 		}
 
-		public float FractionComplete { get { return ticks * 1f / delay; } }
+		public float FractionComplete
+		{
+			get
+			{
+				return skipAscent ? (ticks - turn) * 1f / turn : ticks * 1f / delay;
```

Comment:
The `skipAscent` case here doesn't look right.  I would expect `(ticks - turn) * 1f / (delay - turn)`.